extractor/extraction_containers.cpp

```diff
@@ -464,7 +469,7 @@ void ExtractionContainers::PrepareEdges(lua_State *segment_state)
 
 void ExtractionContainers::WriteEdges(std::ofstream& file_out_stream) const
 {
-    std::cout << "[extractor] Writing used egdes       ... " << std::flush;
+    std::cout << "[extractor] Writing used edges       ... " << std::flush;
     TIMER_START(write_edges);
     // Traverse list of edges and nodes in parallel and set target coord
     unsigned number_of_used_edges = 0;
```

Comment:
would prefer a 64 bit type here for the moment when we actually have more than 2^32 edges, or at least an assert to make sure we do not miss that type.